aip/general/0131.md

```diff
@@ -20,9 +20,10 @@ resource.
 
 ## Guidance
 
-APIs **should** generally provide a get method for resources unless it is not
-valuable for users to do so. The purpose of the get method is to return data
-from a single resource.
+APIs **must** provide a `Get` method *for resources* and **should** do so otherwise,
+unless there is a strong reason not to do so. An example of such, would be if there
+is sensitive data that would be included in the response. The purpose of the `Get` method
```

Comment:
Are we capitalizing `get`?

Reply:
I prefer it as long as it has backticks. It is a proper thing, imho.

Reply:
Precedent AFAICT is "`Get`" and "get". I think we should try to always use `Get` in AIPs to make it explicit that we're talking about a method. In aipdesignbook I try to use "Get" throughout to clarify.